AnnoDesigner.Core/Models/SerializableColor.cs

```diff
@@ -65,5 +65,30 @@ public string ToHexString()
         {
             return $"#{A:X2}{R:X2}{G:X2}{B:X2}";
         }
+
+        public static bool operator ==(SerializableColor left, SerializableColor right)
+        {
+            return Equals(left, right);
+        }
+
+        public static bool operator !=(SerializableColor left, SerializableColor right)
+        {
+            return !Equals(left, right);
+        }
+
+        public override bool Equals(object obj)
+        {
+            return (obj is SerializableColor color) && Equals(color);
+        }
+
+        public bool Equals(SerializableColor other)
+        {
+            return (A, R, G, B) == (other.A, other.R, other.G, other.B);
```

Comment:
As far as I know: No.
`SerializableColor` is a `struct` and not a `class`. `Structs` can never be `null`.

Some explanation I found:
>The Object.ReferenceEquals(...) method is used to determine if two objects are the same instance. Since the method takes two objects as parameters, value types will automatically be boxed in different objects when calling the method. Hence, the method ReferenceEquals(..) will always return false when comparing value type expressions, including struct values. As such, this kind of check is at best useless and at worst erroneous.

A very good blog post about Equality on structs:
https://montemagno.com/optimizing-c-struct-equality-with-iequatable/